### PR TITLE
Add initial implementation

### PR DESCRIPTION
**Pull request by gndelia**
This PR adds the initial implementation exposing a JSON file with the social links.

~It depends on https://github.com/hemilabs/actions/pull/6 being merged to use the actions~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "hemi-socials",
+  "version": "1.0.0",
+  "description": "A repository containing the official list of social links for Hemilabs",
+  "keywords": [
+    "hemilabs"
```
Comment (gabmontes) [inline package.json]: Should we add other keywords, like "discord", "linkedin", "url", "socials", etc?

File: lint-staged.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const formatFiles = "prettier --ignore-unknown --write";
+const sortPackageJson = "better-sort-package-json";
+
+export default {
+  "socials.json|*.{js,md,ts,yml}": [formatFiles],
```
Comment (gabmontes) [inline lint-staged.config.js]: I think that would work better:

```suggestion
  "!(package.json)": [formatFiles],
```

Comment (gndelia) [reply lint-staged.config.js]: Done here [b75e757](https://github.com/hemilabs/hemi-socials/pull/1/commits/b75e7578cc8cf067ae4034f8b8e74eed54d148ad)

File: package.json (hunk 1 above)
Comment (gndelia) [reply package.json]: I added them, though I am not entirely sure how useful these are 😆 
[64c78d9](https://github.com/hemilabs/hemi-socials/pull/1/commits/64c78d9118d3b27a35ef1f8d3be6ff7e113ffc8e)

Comment (gabmontes) [reply package.json]: The keywords are used in sites like npms.io. I think npmjs.com also uses the keywords in its own search.

**Review by gabmontes — DISMISSED** [review]

**Review by gabmontes — APPROVED** [review]

**Review by ArturDolzan — APPROVED** [review]

**Pull request merged**